tools/bigquery-to-xml/bq_to_xml.py

```diff
@@ -0,0 +1,105 @@
+"""
+   Copyright 2020 Google LLC
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+"""
+
+import os
+import xml.etree.ElementTree as ET
+from xml.dom.minidom import parseString
+import lxml.etree as etree
+
+from json2xml import json2xml
+from google.cloud import bigquery
+
+def bigquery_to_xml(query, custom_root_node=False):
+    """ Executes a BQ query and returns the output as an XML string.
+
+    Args:
+      query: The BigQuery query to execute
+      custom_root_node: A string that serves as the root tag for the report. 
+        For example, "sample" would return <sample>[query results]</sample>
+
+      Returns:
+        An XML string containing the BigQuery result.
+    """
+
+    # print("Installing python packages")
+    # os.system("pip3 install json2xml google-cloud-bigquery lxml dicttoxml")
+
+    # print("Setting Google App Credentials")
+    # os.system('export GOOGLE_APPLICATION_CREDENTIALS="/path/to/credentials/file.json"')
```

Comment:
Do you need this ? or assume ADC is already configured ?

Reply:
let's assume that it's configured; removed